spec/book_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/book'
+
+describe Books do
+  books = [{ title: 'managment101', author: 'rich' }, { title: 'cooking101', author: 'fat'}]
+
+
+  it 'constract of empty array at start' do
+   Books = []
```

Comment:
Style/ConstantName: Use SCREAMING_SNAKE_CASE for constants. (https://github.com/bbatsov/ruby-style-guide#screaming-snake-case)<br>Style/MutableConstant: Freeze mutable objects assigned to constants.